python/ray/serve/_private/constants.py

```diff
@@ -228,6 +228,8 @@
 )
 # Serve gauge metric set period.
 RAY_SERVE_GAUGE_METRIC_SET_PERIOD_S = 1
+# How often autoscaling metrics are recorded on Serve replicas.
+RAY_SERVE_REPLICA_AUTOSCALING_METRIC_RECORD_PERIOD_S = 0.5
```

Comment:
What was the previous interval at which we pushed the metrics to the controller? Is this changed? If so, would prefer to make that a separate PR (and test it independently).

Reply:
There are now two separate intervals: one at which we record data for aggregation, and one at which we push the aggregated results to the controller (previously, these were the same). `RAY_SERVE_REPLICA_AUTOSCALING_METRIC_RECORD_PERIOD_S` is now 0.5, but the interval at which we push metrics to the controller hasn't changed - it's default is 10 seconds, and it's tunable by the user.

Reply:
Got it, that's helpful 👍